docs/integrations/webhooks/inngest.mdx

```diff
@@ -0,0 +1,215 @@
+/>
+
+
+You'll be redirected to the new endpoint. In your Inngest dashboard, you will see a new webhook created in your account's [production environment](https://app.inngest.com/env/production/manage/webhooks).
+
+
+## Viewing webhook events within Inngest
+
+After setup, as webhook events are sent from Clerk to Inngest, new `clerk `events will appear  in your Inngest dashboard. Event names will be the prefixed `clerk/` followed by the event name. See [webhook events](/docs/integrations/webhooks#supported-webhook-events) for a full list.
+
+<Images
+  width={1052}
+  height={860}
+  src="/docs/images/integrations/inngest/inngest-clerk-events.webp"
+  alt="The Events page in the Inngest Dashboard showing a list of Clerk events."
+/>
+
+## Creating a function to sync a new user to a database
+
+With Inngest already set up in your codebase, you can now use these events as triggers for your functions.
+
+The code below features the `inngest.createFunction` method, which inserts a new user into the database. It will be triggered whenever `clerk/user.created` event occurs.
+
+
+```typescript
+const syncUser = inngest.createFunction(
+ { id: 'sync-user-from-clerk' },
+ { event: 'clerk/user.created' }, // ← This is the function's triggering event
+ async ({ event }) => {
+   const user = event.data; // The event payload's data will be the Clerk User json object
+   const { id, first_name, last_name } = user;
+   const email = user.email_addresses.find(e =>
+     e.id === user.primary_email_address_id
+   ).email
+   await database.users.insert({ id, email, first_name, last_name })
+ }
+)
+```
+
+
+The `event` object contains all of the relevant data for the event. The `event.data` will match the `data` object from the standard Clerk webhook [payload structure](/docs/integrations/webhooks#payload-structure). With this `clerk/user.created` event, the `event.data` will be a Clerk User json object.
+
+As you can see, you can choose which events you want to handle with each function. You might write a separate function for `clerk/user.updated` and `clerk/user.deleted` handling the entire lifecycle end to end.
+
+Note that multiple functions can also listen to the same event. This pattern is called “[fan-out](https://www.inngest.com/docs/guides/fan-out-jobs).”
+
+
+## Creating a function to send a welcome email
+
+Often, applications need to perform additional tasks when a new user is created, like send a welcome email with tips and useful information.
+
+While it is possible to add this logic at the end of your sync function as seen in the [previous section](/docs/integrations/webhooks/inngest#creating-a-function-to-sync-a-new-user-to-a-database), it's better to decouple logic into different functions so issues with one task do not affect the other ones. For example, if your email fails to send, it should not affect adding the user to an email list.
```

Comment:
@royanger let me know if this paragraph is now clearer 🙏

Reply:
Looks great! Feel free to resolve this once you see!